pkg/virt-operator/creation/components/deployments.go

```diff
@@ -481,6 +481,27 @@ func NewOperatorDeployment(namespace string, repository string, imagePrefix stri
 					Name: name,
 				},
 				Spec: corev1.PodSpec{
+					Affinity: &corev1.Affinity{
+						PodAntiAffinity: &corev1.PodAntiAffinity{
+							PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
+								{
+									Weight: 1,
+									PodAffinityTerm: corev1.PodAffinityTerm{
+										LabelSelector: &metav1.LabelSelector{
+											MatchExpressions: []metav1.LabelSelectorRequirement{
+												{
+													Key:      "kubevirt.io",
+													Operator: metav1.LabelSelectorOpIn,
+													Values:   []string{"virt-operator"},
```

Comment:
Maybe it's better to use the `name` var here?

Reply:
I am not sure since they are different types. Name is a string since Values requires a []string.

I can easily use split:

```                                                                                                       Values:   strings.Split(name, " "),
```

However, I do not think it makes things clearer. And on the other, what will happen if the name contains a space...

Reply:
What I mean is: `[]string{name}`
The thing is, that you specify affinity with the label hard-coded here while it's controlled by the `name` parameter. So is the `kubevirt.io` label which you used as a key... you probably should use `virtv1.AppLabel` instead.

Reply:
Ok, understood. Yes, agree, make sense to use the virtv1.AppLabel.

Thanks for your input @danielBelenky.